robots:
  a_bot:
    default_gripper: a_bot_gripper
  b_bot:
    default_gripper: b_bot_gripper

grippers:
  a_bot_gripper:
    type: RobotiqGripper
    args:
      prefix: a_bot_gripper_
      force: 5.0
      velocity: 0.1
      timeout: 6.0
  b_bot_gripper:
    type: SuctionGripper
    args:
      name: b_bot_gripper
      base_link: b_bot_gripper_base_link
      tip_link: b_bot_gripper_pad_link
      eject: false
      timeout: 2.0

cameras:
  a_phoxi_m_camera: PhoXiCamera
#  a_bot_inside_camera: RealSenseCamera
#  a_bot_outside_camera: RealSenseCamera
